Description: Change the dependencies to use the Gradle plugin built in the bootstrap phase
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/settings.gradle
+++ b/settings.gradle
@@ -9,11 +9,13 @@
   repositories {
     mavenCentral()
     maven {
-      url uri(bnd_repourl)
+      url '/usr/share/maven-repo'
     }
   }
   dependencies {
-    classpath bnd_plugin
+    classpath fileTree(dir: 'bootstrap', include: ['*.jar'])
+    classpath 'org.osgi:osgi.cmpn:debian'
+    classpath 'org.tukaani:xz:debian'
   }
   /* Since the files in the repository change with each build, we need to recheck for changes */
   configurations.classpath {
@@ -33,8 +35,11 @@
   def bndPlugin = files(configurations.classpath.files)
   gradle.rootProject {
     buildscript {
+      repositories { maven { url 'file:///usr/share/maven-repo' } }
       dependencies {
-        classpath bndPlugin
+        classpath fileTree(dir: 'bootstrap', include: ['*.jar'])
+        classpath 'org.osgi:osgi.cmpn:debian'
+        classpath 'org.tukaani:xz:debian'
       }
     }
   }
